### ISSUE TITLE
meta schema is wrongly applied to meta/main.yml in ansible-test integration tests

### ISSUE THREAD

**Issue by felixfontein** [ansible/ansible-lint#2155]
##### Summary
When linting felixfontein.acme with the current schemas, I get a lot of errors such as:
```
schema: 'setup_acme' is not of type 'object' (schema[meta])
tests/integration/targets/role_account_key_rollover-sops/meta/main.yml:1  Returned errors will not include exact line numbers, but they will mention
the schema name being used as a tag, like ``playbook-schema``,
``tasks-schema``.

This rule is not skippable and stops further processing of the file.

Schema bugs should be reported towards (https://github.com/ansible/schemas) project instead of ansible-lint.

If incorrect schema was picked, you might want to either:

* move the file to standard location, so its file is detected correctly.
* use ``kinds:`` option in linter config to help it pick correct file type.
```
Apparently ansible-lint wants to force me to include the full `galaxy_info:` for all integration test roles (for ansible-test). That doesn't make any sense.

(I guess that a special copy of the meta schema needs to be used for ansible-test integration tests that allows `galaxy_info` to be omitted. Or a copy that simply does not allow `galaxy_info`; that doesn't really make sense anyway.)

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

ansible-core devel and ansible-lint main branch

##### STEPS TO REPRODUCE

Run ansible-lint on checkout of https://github.com/felixfontein/ansible-acme/

**Comment by felixfontein** [comment]
Also see https://github.com/felixfontein/ansible-acme/runs/6476807468?check_suite_focus=true. Some of the errors are fixed by https://github.com/ansible/schemas/pull/275, the others are caused by this.

**Comment by ssbarnea** [comment]
How about excluding the test folder from linting?

@cidrblock Do you think we should make galaxy_info block optional in meta schema?

**Comment by ssbarnea** [comment]
@felixfontein I do not think that incorrect schema was picked as I know only one meta schema that applies to roles. Still, I can understand that probably we do not want to enforce every one that creates a `meta/main.yml` to add the galaxy_info block inside it.

**Comment by felixfontein** [comment]
#2156 has a potential fix
